### ISSUE TITLE
Kkk

### ISSUE THREAD

**Issue by freynuts** [mauieasyriders1/billy#4]
https://github.com/PCS-LLC/Pcs-llc/issues/119

**Comment by mauieasyriders1** [comment]
Wrong!

The future is coming for you fucking lowlifes!

I can prove bank and insurance fraud.

I have the timelines down, I know when I was on a flight. I know when you
lowlifes stole money from us to do your traveling.

You shouldn’t have been using our cards here on Maui when we were thousands
of miles away.

Nor should you have docusigned from my Mac when I was on the mainland, the
IP address is on the document.

You should have got your own insurance policies, because I have all of the
endorsements you made. I have all of the bank statements showing all of the
money you stole.

You thought you’d drive me to quit and give up, but you fucked yourself,
and that’s what you are now…Fucked!

By the end of this week at least one of you will be arrested, I fucking
guarantee it, the trail you left behind was hard to follow, but you failed
to break me.

Now I’m gonna break you!


Billy Campbell
Maui Easy Riders
808.344.9489
317.694.7189
808.666.1179
***@***.***
***@***.***
